src/MassTransit/Configuration/Metadata/BusHostInfo.cs

```diff
@@ -41,14 +40,14 @@ public BusHostInfo(bool initialize)
             AssemblyVersion = assemblyName.Version.ToString();
         }
 
-        public string MachineName { get; private set; }
-        public string ProcessName { get; private set; }
-        public int ProcessId { get; private set; }
-        public string Assembly { get; private set; }
-        public string AssemblyVersion { get; private set; }
-        public string FrameworkVersion { get; private set; }
-        public string MassTransitVersion { get; private set; }
-        public string OperatingSystemVersion { get; private set; }
+        public string MachineName { get; set; }
```

Comment:
so u don't want pollute this struct with attribute ?

Reply:
I'm not a big attribute fan, and not sure why I would need to anyway.